server/api.js

```diff
@@ -29,3 +32,23 @@ app.post("/mine", (req, res) => {
 app.listen(HTTP_PORT, () => {
   console.log(`listening on port ${HTTP_PORT}`);
 });
+
+app.post("/transact", (req, res) =>{
+  try{
+    const wallet=new Wallet(req.body.senderPrivateKey);
+    const trans=new Transaction();
+    trans.newTransaction(wallet, req.body.recipientPublicKey, req.body.amount);
+    const bc=Blockchain(Block.genesis());
```

Comment:
first it should be `new Blockchain(...)` and also it should not be initalized to genesis block. It should query the MySQL database and initalize the `Blockchain` object with the data fetched from the db